parlai/core/torch_generator_agent.py

```diff
@@ -967,7 +975,13 @@ def _get_context(self, batch, batch_idx):
 
         Intentionally overridable for more complex model histories.
         """
-        return batch.text_vec[batch_idx]
+        ctxt = batch.text_vec[batch_idx]
+        if self.beam_block_full_context:
+            full_ctxt = batch.observations[batch_idx].get('full_text_vec', ctxt)
+            if not isinstance(full_ctxt, torch.Tensor):
```

Comment:
hmm - should it be `if not isinstance(full_ctxt, torch.LongTensor):` instead?

Reply:
sure, it's just a check that it's not a `List`; will update